components/PoeModal.tsx

```diff
@@ -631,7 +677,7 @@ const PoeModal = () => {
                               state.contracts[address].version,
                               address
                             );
-                            const timeoutAndHash =
+                            let timeoutAndHash =
```

Comment:
I don't see `timeoutAndHash` being reassigned, why changing it to let?

Reply:
Done, I was confused about the state with same name haha